index.html

```diff
@@ -281,6 +281,49 @@ <h3>
                 </div>
             </div>
         </div>
+        <footer class="contact-form">
+            <div class="contact-heading">
+                <h2 class="contact-h2">
+                    I'm always<br>
+                    interested in<br>
+                    hearing about<br>
+                    new projects, so<br>
+                    if you'd like to<br>
+                    chat please get<br>
+                    in touch.
+                </h2>
+            </div>
+            <form action="https://formspree.io/f/xnqrnjgr" class="contact-details" method="post">
+                <ul class="contact-list">
+                    <li>
+                        <input type="text" class="contact-1" maxlength="30" placeholder="Full name" required>
+                    </li>
+                    <li>
+                        <input type="email" class="contact-2" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <input type="text" class="contact-3" maxlength="500" placeholder="Enter text here" required>
```

Comment:
- [ ] Kindly be noticed that the message should be `textarea` tag not `input` tag.
```html
<textarea class="contact-3" maxlength="500" placeholder="Enter text here" required></textarea>
```